### DISCUSSION TITLE
Strange animated object on the map [video]

### DISCUSSION THREAD

**Discussion by Botrops1** [streetcomplete/StreetComplete#5093] [General]
https://github.com/streetcomplete/StreetComplete/assets/133165279/551818e0-334a-47bb-80de-2da9f23a6579

Hi all,
Just discovered a strange animated object on the map. Not sure if it was there before.
It has some Ingress flavor, tbh
Do you know what is it, should I report a bug?
Thanks

**Comment by FloEdelmann** [comment]
That's a small body of water, see https://github.com/streetcomplete/StreetComplete/issues/4542#issuecomment-1287796776
